subprojects/core/src/main/java/org/gradle/api/artifacts/LenientConfiguration.java

```diff
@@ -35,6 +35,13 @@
     public Set<ResolvedDependency> getFirstLevelModuleDependencies(Spec<? super Dependency> dependencySpec);
 
     /**
+     * Returns all successfully resolved dependencies.
+     *
+     * @return all resolved dependencies
```

Comment:
Can you add a `@since` annotation to indicate the Gradle version that introduces the new method?